git_machete/client.py

```diff
@@ -2021,7 +2021,11 @@ def create_github_pr(
         debug(f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
         debug(f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
 
-        title: str = self.__git.get_commit_information("subject").strip()
+        fork_point = self.fork_point(head, True, opt_no_detect_squash_merges=False)
+        if not fork_point:
+            raise MacheteException(f"Could not find a fork-point for branch {head}.")
+        commits: List[Hash_ShortHash_Message] = self.__git.get_commits_between(fork_point, head)
+        _, _, title = commits[0]
```

Comment:
Hmmm in case `commits` is empty (which might happen! when no unique commit committed on the current branch has been detected &mdash; e.g. that's a brand new branch)... this will fail! in such case, I'd fall back to the current commit (as it used to be before your fix)

Reply:
Ouch! good spot ;)